Tlane77/Asclepius
# Tl View, Add and Initial Edit Files with Registration and Login

# Description
As the Asclepius product owner, I would like all users to be authenticated in order to perform any activity in the system so that the system will be able to record which user created condition, comment, etc.

Given an unauthenticated user is in the Asclepius application
When they click any link
Then they should be prompted to login using their email address

Given an unauthenticated user is viewing the login form
When they enter an email address that matches an existing User Profile
Then they should be authenticated into the system
And they should be directed to the application home page

Given an unauthenticated user is viewing the login form
When they enter an email address that does not match an existing User Profile
Then an error message should be displayed
And the user should be given another change to enter a valid email address

#### Styling
Added 10% margins to either side of window for all views to create a whitespace border.

# Ticket Reference

> [#1 - Basic Auticentation](https://github.com/nss-day-cohort-41/Asclepius/compare/tlViewConditions/1)
> [#3  -View all Conditions](https://github.com/nss-day-cohort-41/Asclepius/compare/tlViewConditions/3)
> [#4  - Create/Add a Condition](https://github.com/nss-day-cohort-41/Asclepius/compare/tlViewConditions/4)

## Type of change


# Testing Instructions
- Run both files in the SQL folder. (CREATE and SEED DATA
- Run a SQL query to add a user to the UserProfile table with an id of 1 and a display name "techMe" and email "tech3@health.com"
- Start program
- Go to localhost:3000 in your browser.
- Login with "tech3@health.com"
- Navigate to the "Conditons" link in the navbar.
-Visually Confirm List of Conditions is Displayed.
-Click Add and Look at the Add Form  Make sure all fields are filled then click save button.
-Verify Data Has been saved to the database.
- Next, click the logout button on the navbar.
- Enter tech3@health.com into the email field and click "Login".
-Confirm  add is populating in list.

# Checklist:
